pmm/pmm2-api-tests.groovy

```diff
@@ -73,72 +69,63 @@ pipeline {
                 sh 'git checkout ' + env.GIT_COMMIT_HASH
             }
         }
-        stage('Start staging') {
-            steps {
+        stage('API Tests Setup')
+        {
+            steps{
+                sh '''
+                    sudo curl -L https://github.com/docker/compose/releases/download/1.29.0/docker-compose-`uname -s`-`uname -m` | sudo tee /usr/local/bin/docker-compose > /dev/null
+                    sudo chmod +x /usr/local/bin/docker-compose
+                    sudo ln -sfn /usr/local/bin/docker-compose /usr/bin/docker-compose
```

Comment:
Why not put it straight to `/usr/bin/docker-compose` instead of linking?

Reply:
can do that, will implement in next push and test it

Reply:
Will try and push in the next one

Reply:
I have pushed this change, thank you